doc/Command-Reference.md

```diff
@@ -1,3712 +1,3734 @@
-
-**config ecn**  
-
-This command configures the possible fields in a particular WRED profile that is specified using "-profile <profilename>" argument. 
-The list of the WRED profile fields that are configurable is listed in the below "Usage".
-
-  - Usage:   
-    config ecn [OPTIONS]
-	
-  ```
-    ECN Config OPTIONS:
-	  -profile <profile_name>       Profile name  [required] - Even though the profile_name is specified as optional parameter, it is a mandatory parameter.
-	  -rmax <red threshold max>     Set red max threshold
-	  -rmin <red threshold min>     Set red min threshold
-	  -ymax <yellow threshold max>  Set yellow max threshold
-	  -ymin <yellow threshold min>  Set yellow min threshold
-	  -gmax <green threshold max>   Set green max threshold
-	  -gmin <green threshold min>   Set green min threshold
-	  -v, --verbose                 Enable verbose output
-	  --help                        Show this message and exit.
-  ```
- 
-- Example:
-  ``` 
-  root@T1-2:~# config ecn -profile wredprofileabcd -rmax 100
-     This command configures the "red max threshold" for the WRED profile name "wredprofileabcd". It will create the WRED profile if it does not exist.
-  ```
-
-Go Back To [Beginning of the document](#SONiC-COMMAND-LINE-INTERFACE-GUIDE) or [Beginning of this section](#ECN-Configuration-And-Show-Commands)
-
-# Interface Configuration And Show-Commands
-
-## Interface Show Commands
-
-This sub-section lists all the possible show commands for the interfaces available in the device. Following example gives the list of possible shows on interfaces.
-Subsequent pages explain each of these commands in detail.
-
-- Example:
-  ```
-  user@debug:~$ show interfaces -?
-
-  Show details of the network interfaces
-
-  Options:
-    -?, -h, --help  Show this message and exit.
-
-  Commands:
-  counters     Show interface counters
-  description  Show interface status, protocol and...
-  naming_mode  Show interface naming_mode status
-  neighbor     Show neighbor related information
-  portchannel  Show PortChannel information
-  status       Show Interface status information
-  transceiver  Show SFP Transceiver information
-  ```
-
-**show interfaces counters**  
-
-This show command displays packet counters for all interfaces since the last time the counters were cleared. There is no facility to display counters for one specific interface. Optional argument "-a" does not have any significance in this command.
-Optional argument "-c" can be used to clear the counters for all interfaces.
-Optional argument "-p" specify a period (in seconds) with which to gather counters over.
-
-  - Usage:  
-    show interfaces counters [OPTIONS]  
-      OPTIONS:  
-      -a, --printall  
-      -c, --clear  
-      -p, --period TEXT
-
-
-- Example:
-  ```
-  admin@sonic:~$ show interfaces counters
-        IFACE    STATE            RX_OK       RX_BPS    RX_UTIL    RX_ERR    RX_DRP    RX_OVR            TX_OK       TX_BPS    TX_UTI
```

Comment:
why this file got changed entirely?

Reply:
'set filetype=unix'. This was done in a separate PR. Now I rebased on master